docs/testing/debug-add-ins-using-devtools-edge-chromium.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Debug add-ins using developer tools for Microsoft Edge WebView2
+description: 'Debug add-ins using the developer tools in Microsoft Edge WebView2 (Chromium-based).'
+ms.date: 11/09/2021
+ms.localizationpriority: medium
+---
+
+# Debug add-ins using developer tools in Microsoft Edge (Chromium-based)
+
+This article shows how to debug the client-side code (JavaScript or TypeScript) of your add-in when the following conditions are met.
+
+- You cannot, or don't wish to, debug using tools built into your IDE; or you are encountering a problem that only occurs when the add-in is run outside the IDE.
+- Your computer is using a combination of Windows and Office versions that use the Edge (Chromium-based) webview control, WebView2.
+
+> [!TIP]
+> For information about debugging with Edge WebView2 (Chromium-based) inside Visual Studio Code, see [Debug add-ins on Windows using Visual Studio Code and Microsoft Edge WebView2 (Chromium-based)](debug-desktop-using-edge-chromium.md).
+
+To determine which browser you're using, see [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md).
+
+> [!TIP]
+> [!INCLUDE[Identify the webview through the add-in UI](../includes/identify-webview-in-ui.md)]
+
+## Debug a task pane add-in using Microsoft Edge (Chromium-based) developer tools
+
+> [!NOTE]
+> If your add-in has an [add-in command](../design/add-in-commands.md) that executes a function, the function runs in a hidden browser process that the Microsoft Edge (Chromium-based) developer tools cannot be launched from, so the technique described in this article cannot be used to debug code in the function.
+
+1. [Sideload](create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md) and run the add-in.
+1. Run the Microsoft Edge (Chromium-based) developer tools by one of these methods:
+
+   - Be sure the add-in's task pane has focus and press **Ctrl-Shift-I**.
+   - Right-click the task pane to open the context menu, or open the [personality menu](../design/task-pane-add-ins.md#personality-menu), and select **Inspect**.
```

Comment:
I see inspect in the right click menu, but I see "Attach Debugger" in the personality menu.  The both (along with the shortcut) open up the developer tools as expected.

Reply:
Good catch. Fixed.